### PR TITLE
ONL-4676 menu migration

### PR DESCRIPTION
**Pull request by jaimeperezpedra**
**Layout Menu migration.**

There is some unit test that we can fix it(because they are using classes) once we do the migration of the [navigation-link](https://fxsolutions.atlassian.net/browse/ONL-4678).

### CODE REVIEW

File: src/components/ec-menu/ec-menu.vue [hunk 1]
```diff
@@ -50,34 +53,29 @@ export default {
...
-  margin: 0;
-  display: flex;
-  flex-direction: column;
-  flex-wrap: wrap;
-  align-items: flex-start;
-  justify-content: flex-start;
+  @apply tw-list-none;
+  @apply tw-p-0;
+  @apply tw-m-0;
+  @apply tw-flex tw-flex-row tw-flex-wrap;
```
Comment (matallo) [inline src/components/ec-menu/ec-menu.vue:61]: wasn't it column? 🤔

Comment (jaimeperezpedra) [reply src/components/ec-menu/ec-menu.vue]: done

File: src/components/ec-menu/ec-menu.vue [hunk 2]
```diff
@@ -50,34 +53,29 @@ export default {
...
+  @apply tw-m-0;
+  @apply tw-flex tw-flex-column tw-flex-wrap;
+  @apply tw-items-start tw-justify-start;
 
   &--horizontal {
-    flex-direction: row;
-    justify-content: center;
-    align-items: center;
+    @apply tw-flex-row;
+    @apply tw-items-start tw-justify-start;
```
Comment (mcibique) [inline src/components/ec-menu/ec-menu.vue:66]: I think both should be `*-center`

Comment (jaimeperezpedra) [reply src/components/ec-menu/ec-menu.vue]: done

File: src/components/ec-menu/ec-menu.story.js [hunk 3]
```diff
@@ -39,10 +39,10 @@ stories
       },
     },
     template: `
-      <div style="margin: 0 auto;" :style="{ maxWidth: width + 'px' }">
+      <div class="tw-my-auto tw-mx-auto" :style="{ maxWidth: width + 'px' }">
```
Comment (jmesao) [inline src/components/ec-menu/ec-menu.story.js:42]: my is 0 no auto, continue having the same behavior?

Comment (jaimeperezpedra) [reply src/components/ec-menu/ec-menu.story.js:42]: Yes it has the same behaviour, anyway I'll fix it

**Pull request merged**